src/Native/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -307,6 +321,7 @@ bool DoTheStep(uintptr_t pc, UnwindInfoSections uwInfoSections, REGDISPLAY *regs
     }
 
     regs->pIP = PTR_PCODE(regs->SP - sizeof(TADDR));
+#endif
```

Comment:
Add `#else` path with `PORTABILITY_ASSERT("DoTheStep");` to make it easy to see if somebody hits this unimplemented path?